R/tar-terra-rast.R

```diff
@@ -175,6 +182,38 @@ tar_rast_write <- function(filetype, gdal, preserve_metadata) {
             #move the zip file to the expected place
             file.rename(file.path(tmp, basename(path)), path)
         },
+        gdalraster_sozip = function(object, path) {
+
+            tmp <- withr::local_tempdir()
+
+            dir.create(file.path(tmp, dirname(path)), recursive = TRUE)
```

Comment:
Suggest moving `file.path(tmp, dirname(path))` out into a separate variable, since it gets referred to a few times later on in this function

Reply:
ok this is done. Sorry for some commit noise on this, made a mistake that broke tests, but I think it is resolved now

Reply:
See also #127 -  the fix(es) for that issue are maybe also required here.